iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -69,8 +69,10 @@ constexpr uint32_t MAX_SUBSCRIBER_QUEUE_CAPACITY = MAX_CHUNKS_HELD_PER_SUBSCRIBE
 // 1x publisherPort mempool introspection
 // 1x publisherPort process introspection
 // 3x publisherPort port introspection
-// 1x publisherPort service registry
-constexpr uint32_t PUBLISHERS_RESERVED_FOR_INTROSPECTION_AND_SERVICE_REGISTRY = 6;
```

Comment:
Not for now, but in the future it may be useful to break this down into headers for the configuration of individual components (which are included by the main config). Otherwise we end up with one monolithic config file. This would be just for logical grouping.

Resolve me.